### ISSUE TITLE
[BUG] application audio "app:" doesn't work with audio track name "/"

### ISSUE THREAD

**Issue by GenocideStomper** [dec05eba/gpu-screen-recorder-issues#71]
**Describe the bug**
Application audio using `app:` or `app-inverse:` doesn't seem to work if there's a name prefixed to the audio track using `/`.

**To Reproduce**
Running with an argument such as `-a 'audio-track-name/app:Chromium'` outputs the following error before exiting:

> Error: Audio input device 'app:Chromium' is not a valid audio device, expected one of:
> < list of audio devices >

Running with just `-a 'app:Chromium'` works.

**Desktop (please complete the following information):**
 - X11 or Wayland: X11
- Desktop environment/Window Manager: KDE Plasma 6
 - Distro: endeavourOS
 - GPU: Nvidia GTX 1080
 - Version (latest AUR): r909.8644c72-1  / 4.3.3

**Additional context**
Not sure if this is intended behavior and naming tracks is only intended for audio devices and not audio apps.

**Comment by dec05eba** [comment]
It's intentional. But also "/" is not for setting the audio track name, it's for setting the name of the recording node in pulseaudio/pipewire, but this option is not really needed anymore. I'll change "/" so it can be used to set audio track name instead (and work when using app:/app-inverse:.

**Comment by GenocideStomper** [comment]
Thank you very much!

**Comment by GenocideStomper** [comment]
> It also works with merged audio, for example: `-a 'audio-track-name/app:Chromium/device:default_input'`. The change has been pushed to aur as well.

I'm just re-reading your comment, with merged audio don't you need the `|` pipe character between your example of app:Chromium and device:default_input? I assume it was just a typo in your example?

**Comment by dec05eba** [comment]
You are correct, it was a typo, fixed. `gpu-screen-recorder --help` shows the correct format with |.
